introduction/examples/thompson/erastothenes.py

```diff
@@ -0,0 +1,33 @@
+# Function returns a list of prime numbers from 2 to n
+def eratosthenes(n):
+    if isinstance(n, int):
+        if n >= 2:
+            numbers = [False, False]
+            primes = []
+            i = n - 1
+            while i:
```

Comment:
`while i` wiem, że wiesz, że wyjdziemy z pętli gdy `i == 0` ale osoba nie znająca pythona może o tym nie wiedzieć. Popraw warunek (lub przejdź wgl na pętlę `for`)